opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -620,3 +620,27 @@ excludeServiceMonitorsFromRemotePoller=DHCP,NSClient,RadiusAuth,XMP
 #   placeholder
 #   backshift
 #org.opennms.web.graphs.engine=png
+
+# ###### GRAFANA BOX ######
+# This setting controls whether a grafana box showing the available dashboards is
+# placed on the landing page. The two valid options for this are 'true' or 'false'
+# which is the default value.
+#org.opennms.grafanaBox.show=false
+#
+# If the box is enabled you also need to specify hostname of the grafana server
+#org.opennms.grafanaBox.hostname=localhost
+#
+# And also the port of the grafana server
+#org.opennms.grafanaBox.port=3000
+#
+# The API key is needed for the ReST calls to work
+#org.opennms.grafanaBox.apiKey=
+#
+# When a tag is specified only dashboards with this given tag
+# will be displayed. When no tag is given all dashboards will
+# be displayed.
+#org.opennms.grafanaBox.tag=
+#
+# The protocol for the ReST call can also be specified
+#org.opennms.grafanaBox.protocol=http
+
```

Comment:
Currently there is a discussion going on whether to move properties-documentation to it's own doc module.
As Ronny is mostly driving this, can you please clear with him, if you need to create an issue or something for that?